Libbna/CUSTOM-CMS
# #222: Changed the styling for Menu and Block display page

Steps to test:
* Login and goto users display page see the styling 
* Go to block and menu display pages and see if styling for those pages are similar
